### DISCUSSION TITLE
Does Ani-Cli have One Piece dubbed?

### DISCUSSION THREAD

**Discussion by BradyBoo62** [pystardust/ani-cli#1096] [General] [answered]
I searched up One Piece and couldn't find the dubbed episodes. I tried to use --dub but no results were found.

**Comment by CoolnsX** [comment, marked as answer]
```bash
ani-cli --dub one piece
```
![Screenshot_20230423-103750](https://user-images.githubusercontent.com/76195824/233820955-c9241ee3-72dc-4092-92ad-d932dd431419.png)

**Reply by BradyBoo62** [reply]
Thank you so much, you're a lifesaver!
